ci/deploy_app.sh

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env bash
+set -exo pipefail
+
+REGION="${REGION:=europe-west2}"
+CPU="${CPU:=4}"
+MEMORY="${MEMORY:=4G}"
+CONCURRENCY="${CONCURRENCY:=80}"
+CDN_URL="${CDN_URL:=https://cdn.census.gov.uk}"
+NEW_RELIC_APP_NAME="${NEW_RELIC_APP_NAME:=}"
+NEW_RELIC_LICENSE_KEY="${NEW_RELIC_LICENSE_KEY:=}"
+COOKIE_SETTINGS_URL="${COOKIE_SETTINGS_URL:=#}"
+ADDRESS_LOOKUP_API_URL="${ADDRESS_LOOKUP_API_URL:=#}"
```

Comment:
I'm not sure these should default to `#`, runner enables these things based on the variables being set.

Reply:
I removed these values so it defaults to empty now.